### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by veggiemonk**
added the twitter alias of the maintainers

### CODE REVIEW

**Comment by veggiemonk** [comment]
Hey @bbodenmiller, you can add your twitter handle if you want to be part of the experience 😄

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -6,24 +6,28 @@ Please note that this project is released with a [Contributor Code of Conduct](h
...
+- want to change something: Submit a pull request or open an issue
+- want to try something else: Submit a pull request or open an issue
 - don't know what to do: Submit a pull request or open an issue, let us know what's going on.
 
+or you can
+
+#### ping us on twitter:
+
+* [veggiemonk](https://twitter.com/veggiemonk)
+* [vegasbrian](https://twitter.com/idomyowntricks)
```
Comment (vegasbrianc) [inline CONTRIBUTING.md]: Small typo. Twitter name is Idomyowntricks.

**Comment by veggiemonk** [comment]
@vegasbrianc  Sorry about that, I just copy paste it from twitter.

**Comment by veggiemonk** [comment]
@vegasbrianc you can merge it now...

**Pull request merged**